typescript/src/commands/Tools/Dictionary/define.ts

```diff
@@ -12,7 +12,7 @@ import type { TFunction } from 'i18next';
 
 @ApplyOptions<PaginatedMessageCommand.Options>({
 	enabled: envIsDefined('OWLBOT_TOKEN'),
-	aliases: ['definition', 'defination', 'dictionary'],
+	aliases: ['def', 'definition', 'dictionary'],
```

Comment:
```suggestion
	aliases: ['def', 'definition', 'defination', 'dictionary'],
```

The misspelling was there intentionally because it's one of those words that people often misspell.